src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -300,35 +304,37 @@ public static OrganisationDetails getOrganisationDetails(String name, String typ
 
     public static List<UnavailabilityDayOfWeek> getPartyUnavailabilityDayOfWeek() {
         // Not used as of now
-        return null;
+        return Collections.emptyList();
     }
 
     public static List<UnavailabilityDayOfWeek> getDwpUnavailabilityDayOfWeek() {
         // Not used as of now
         return getPartyUnavailabilityDayOfWeek();
     }
 
+    public static List<UnavailabilityRange> getPartyUnavailabilityRangeAllDay(HearingOptions hearingOptions) {
+        List<UnavailabilityRange> partyUnavailabilityRange = getPartyUnavailabilityRange(hearingOptions);
+        partyUnavailabilityRange.forEach(unavailabilityRange -> unavailabilityRange.setUnavailabilityType(ALL_DAY.getLabel()));
```

Comment:
Can't you set this in getPartyUnavailabilityRange so there's no need to iterate again? You won't need this method, then

Reply:
The reason for the split is that getPartyUnavailabilityRange is used by ServiceHearingPartiesMapping which does not have the UnavailabilityType, rather than have a duplicate method for SHV. 
I could add a parameter for UnavailabilityType that is null for SHV, do you think that would be better?